walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java

```diff
@@ -24,7 +24,11 @@
 @WebServlet("/page-views")
 public class PageViewServlet extends HttpServlet {
 
-  private int pageViews = 0;
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
```

Comment:
Where is serialVersionUID used?

Reply:
When I used to run the mvn command in the console, I would get a serialization error. In the .java file, there was a yellow underline under the class name of the program and it said "The serializable class HelloWorldServlet does not declare a static final serialVersionUID field of type long." In the end, I clicked the quick fix option and the mvn command ran without errors.